wherehows-backend/app/controllers/Application.java

```diff
@@ -42,11 +40,13 @@
   private static final String DB_WHEREHOWS_PASSWORD =
       Play.application().configuration().getString("db.wherehows.password");
   private static final String DB_WHEREHOWS_DIALECT = Play.application().configuration().getString("hikaricp.dialect");
+  private static final String DAO_FACTORY_CLASS =
+      Play.application().configuration().getString("dao.factory.class", DaoFactory.class.getCanonicalName());
 
-  public static final EntityManagerFactory entityManagerFactory;
+  private static final EntityManagerFactory ENTITY_MANAGER_FACTORY;
 
   static {
```

Comment:
Let's just do inline initialization instead of static block to improve readability?